packages/analytics-browser/src/browser-client.ts

```diff
@@ -169,12 +171,11 @@ export class AmplitudeBrowser extends AmplitudeCore implements BrowserClient {
     return this.config?.sessionId;
   }
 
-  setSessionId(sessionId: number) {
+  setSessionId(sessionId: number, shouldTrackNewCampaign?: boolean) {
```

Comment:
Do we need to add `shouldTrackNewCampaign?: boolean` to the method signature? We didn't have that before, and would like to avoid interface changes if possible